tests/unit/api/c/source/test_source.cpp

```diff
@@ -40,7 +40,14 @@
 #define TestSource TestSourceStatic
 #endif
 
-#define NUM_TEST_NETINTS 3
+#define NUM_TEST_NETINTS 3u
```

Comment:
Always default to constexpr types instead of #define for constants.

```cpp
constexpr size_t kNumTestNetints = 3u;
// etc...
```

Reply:
https://github.com/ETCLabs/sACN/pull/47/commits/230a18a1140f41ba08a99dce914d2ae08bc49481